### PR TITLE
Add gratia-probe-condor-batch (SOFTWARE-4978)

### PR DESCRIPTION
**Pull request by brianhlin**

### CODE REVIEW

File: rpm/gratia-probe.spec [hunk 1]
```diff
@@ -400,7 +436,7 @@ Requires: htcondor-ce
 License: See LICENSE.
 
 %description htcondor-ce
-The HTCondor-CE probe for the Gratia OSG accounting system.
+The HTCondor-CE probe for the Gratia OSG accounting system. Designed for use with non-HTCondor batch systems.
```
Comment (matyasselmeci) [inline rpm/gratia-probe.spec:442]: This suggests we don't want it running if the batch system is condor?

File: rpm/gratia-probe.spec [hunk 2]
```diff
@@ -287,19 +295,47 @@ fi
...
 # %%{default_prefix}/gratia/common2/probeinput.py
 
+%package condor-batch
+Summary: A probe accounting for pilot jobs in a local HTCondor batch system
+Requires: %{name}-common = %{version}-%{release}
+Requires: condor
+Requires: %{condor_python}
+Requires: htcondor-ce
+Provides: %{name}-condor
+Obsoletes: %{name}-condor
```
Comment (matyasselmeci) [inline rpm/gratia-probe.spec]:
```suggestion
Provides: %{name}-condor = %{version}-%{release}
Obsoletes: %{name}-condor < 2.4.0
```

File: rpm/gratia-probe.spec (hunk 1 above)
Comment (brianhlin) [reply rpm/gratia-probe.spec:442]: Yeah, config and crons would fight

Comment (matyasselmeci) [reply rpm/gratia-probe.spec:442]: wanna add a Conflict?

Comment (brianhlin) [reply rpm/gratia-probe.spec:442]: Done in the latest commit

**Review by matyasselmeci — APPROVED** [review]
LGTM.

**Pull request merged**